packages/graphql/src/schemaTypes/objectTypes/gql-CodeGenResult.ts

```diff
@@ -0,0 +1,49 @@
+import { enumType, objectType } from 'nexus'
+import { FileParts } from '.'
+
+const CodeGenResultStatusEnum = enumType({
+  name: 'CodeGenStatus',
+  members: ['add', 'overwrite', 'skipped'],
+})
+
+const CodeGenGenResultTypeEnum = enumType({
+  name: 'CodeGenGenResultType',
+  members: ['text', 'binary'],
+})
+
+export const CodeGenResult = objectType({
+  name: 'CodeGenResult',
+  description: 'Represents a spec on the file system',
+  node: 'file',
+  definition (t) {
+    t.nonNull.field('status', {
+      type: CodeGenResultStatusEnum,
+    })
+
+    t.nonNull.field('type', {
+      type: CodeGenGenResultTypeEnum,
+    })
+
+    t.nonNull.string('file', {
+      description: 'Absolute path of generated file.',
+    })
+
+    t.nonNull.string('content', {
+      description: 'Content of generated file.',
+    })
+  },
+})
+
+export const CodeGenResultWithFileParts = objectType({
```

Comment:
Same thing here with multiple objectTypes

Reply:
I'm going to keep this one in here since I'm really only extending the type.